inc/class-destination-s3.php

```diff
@@ -159,12 +159,12 @@ class="regular-text"
                                         <label
                                             for="s3base_pathstylebucket">
                                             <input name="s3base_pathstylebucket" type="checkbox"
-                                                   value="<?= !empty(
+                                                   value="1" <?= !empty(
                                                    BackWPup_Option::get(
                                                        $jobid,
                                                        's3base_pathstylebucket'
-                                                   ) ? '1' : ''
-                                                   ) ?>">
+                                                   )
+                                                   ) ? 'checked="checked"' : '' ?>>
```

Comment:
Same as above, should use [checked()](https://developer.wordpress.org/reference/functions/checked/)